### PR TITLE
Finaltouches

### PR DESCRIPTION
**Pull request by BreonKnight**

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -1,11 +1,94 @@
...
+// font-family: 'Faster One', cursive;
+// font-family: 'Gudea', sans-serif; 
+// font-family: 'Monoton', cursive;
+
+// COLORS //
+$violetred: #FA5096;
+$teaRose: #FF8ABA;
+$freeSpeechMagenta: #D83FEF;
+$lavenderIndigo: #AF4DF8;
+$caribianGreen: #26F9B2;
```
Comment (tgaff) [inline app/assets/stylesheets/users.scss]: please rename to `caribbeanGreen`

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -1,2 +1,15 @@
 module UsersHelper
+
+  def user_ud(user)
```
Comment (tgaff) [inline app/helpers/users_helper.rb]: this method name means nothing to the casual observer (me)  please rename it.

Also it's common to name methods that return true/false with a question mark, eg: `def is_empty?`

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+    user == current_user
+  end
+
+  def upvoteCount(user)
```
Comment (tgaff) [inline app/helpers/users_helper.rb]: please use ruby naming conventions: `upvote_count`

File: app/views/static_pages/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
-<div class="jumbotron">
+<div class="jumbotron hero">
 	<div class="center-block">
-		<h1 class="center-text">Fleeking Beauty</h1>
+		<h1 class="text-center h1jumbo">Fleeking Beauty</h1>
```
Comment (tgaff) [inline app/views/static_pages/index.html.erb:3]: I might be wrong, but is `h1jumbo` a real class?

File: app/views/users/index.html.erb [hunk 5]
```diff
@@ -5,9 +5,9 @@
 		<% @users.each do |user| %>
 		<div class="col-md-4">
 			<%= link_to(user) do %>
-			<h1><%= user.userName %></h1>
+			<h3 class="text-center userIndex"><%= user.userName.downcase %></h3>
```
Comment (tgaff) [inline app/views/users/index.html.erb]: ideally define a convention for naming CSS attributes.  I see spinalcase `text-center` and camel case `userIndex`.

File: app/views/users/show.html.erb [hunk 6]
```diff
@@ -1,33 +1,31 @@
...
-	<div class="row">
-		<div class="col-md-12">
-			  <h2 class="text-center"><%= @user.userName %> </h2>
-			  <%= image_tag @user.avatar.url, size: "320x426", class: "img-responsive" %>
-			  <h3><%=@user.first_name%> <%=@user.last_name%></h3>
-			  <% if current_user == @user %>
-			  <%= button_to "Edit", edit_user_path, method: :get, class: "btn btn-default" %>
+<div class="container">
+  <div class="row">
+    <h1 class="text-center"><%= @user.userName %> </h1>
```
Comment (tgaff) [inline app/views/users/show.html.erb:3]: hmm... that doesn't look like a ruby name, does your user model really have a `userName` field?  If so please make a migration to change it to `name`.

File: app/views/users/show.html.erb [hunk 7]
```diff
@@ -1,33 +1,31 @@
...
-</div>
-<div class ="container ">
-	<div class = "row">
-	<br>
-	Total Upvotes: <%= @user.upvotes.count  %> 
-	<br>
-	Total Downvotes: <%= @user.downvotes.count  %> 
-	<br>
-<% @places.each do |place| %>
+        <% if user_ud(@user) %>
```
Comment (tgaff) [inline app/views/users/show.html.erb]: please rename this method

File: config/routes.rb [hunk 8]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
 
-  root to: "users#index"
+  root to: "static_pages#index"
```
Comment (tgaff) [inline config/routes.rb:3]: liking this static pages controller!

**Comment by tgaff** [comment]
Review complete, please fix before merging.  If you push more commits to this branch it will automatically update the PR.

Looks pretty good overall, almost all my commits are about style issues which can be tricky when you're doing JS and Ruby simultaneously.

File: app/views/users/show.html.erb (hunk 6 above)
Comment (BreonKnight) [inline app/views/users/show.html.erb:3]: This is a bug, we will file this under our bugs list

**Pull request closed without merging**